### ISSUE TITLE
when I train the model,why is it always interrupted?

### ISSUE THREAD

**Issue by wxd19961228** [elleryqueenhomels/arbitrary_style_transfer#8]
I downloaded the wikiart and MSCOCO dataset. The numger of them are 80k and 110k respectively. And I preprocess the content images and the style images(wikiart dataset has some pictures that are damaged,but MSCOCO not). So I begin to train the model, but it is always interrupted before ending up with the first epoch.  Can you give me some advice?
The wrong results are:(one epoch should have about 9900steps)

step: 5620,  total loss: 2517.050,  elapsed time: 0:54:17.473793
content loss: 2478.894
style loss  : 3815.609,  weighted style loss: 38.156

Something wrong happens! Current model is saved to <D:\wxd\arbitrary_style_transfer-master_0\tmp_model>

Done training! Elapsed time: 0:54:30.136937
Model is saved to: model_retrain/style_weight_1e-2.ckpt

>>> Successfully! Done all training...

we end at 2019-01-06 10:24:25.703097

**Comment by elleryqueenhomels** [comment]
Hi,
It seems that there are still some damaged images in the wikiart. I update the repo to print out the error message during training. If you could provide more details, it would be great help.
Thanks!

**Comment by wxd19961228** [comment]
I am very thankful that you reply me so soon, and your advice is very useful. I check the wikiart dataset again, and there are still some damaged pictures.
Now I solve that problem. But I have another problem as following:
when I train the model,  I set the style_weight equals 0.01(yours is 2.when I infer on your model, I found that style is toooutstanding, so I decrease it)the content loss always high even though the style loss is really small. 
ps: this is my first time to practice the deep learning, can you give me some advice on how to adjust the parameters to get better result?
and these are my own ideas, I do not know whether they can take effect:
1. enlarge the batchsize(I think after shuffle, small batchsize may result in unstable)
2.change the content loss function. I want to use a more subtle loss just like the style loss function which has four terms(relu1,relu2,relu3,relu4). when I read the original paper, I.don't know why the author takes that content loss(the author doesn't do contrast experiment)

here are my result(only change the style_weight based on your model)
![image](https://user-images.githubusercontent.com/43460042/50736166-f44ead00-11f4-11e9-8c98-ef5f03e9f098.png)


I am looking forward to your reply.
thank you very much

**Comment by elleryqueenhomels** [comment]
It's worth to have a try on your ideas. Especially changing the content loss, it may yield a better model. Regarding on increasing batch size, I don't think it could bring great effect, since our purpose to use mini-batch is to introduce randomness to avoid overfitting. But you could still have a try. Keep in mind, if you are using GPU to train, too large batch size may cause out-of-memory issue.

**Comment by wxd19961228** [comment]
yeah, I think your suggestion is right. And I will have a try. Thank you very much. And I still have the question: when you train the model, the loss is big as mine? The content loss is about 2000-5000 and it seems never converged.

**Comment by wxd19961228** [comment]
yeah, I think your suggestion is right, and I will have a try. But I still have s question: when you train the model, is your content loss as mine? The content loss is about 2000-5000, and it seems never converge.
